packages/dbos-cloud/applications/delete-app.ts

```diff
@@ -1,13 +1,13 @@
 import axios, { AxiosError } from "axios";
 import { isCloudAPIErrorResponse, handleAPIErrors, getCloudCredentials, getLogger, retrieveApplicationName } from "../cloudutils";
 
-export async function deleteApp(host: string): Promise<number> {
+export async function deleteApp(host: string, appName?: string): Promise<number> {
   const logger = getLogger()
   const userCredentials = getCloudCredentials();
   const bearerToken = "Bearer " + userCredentials.token;
 
-  const appName = retrieveApplicationName(logger);
-  if (appName === null) {
+  appName = appName ?? retrieveApplicationName(logger);
+  if (!appName) {
```

Comment:
How does that work from a directory where there is no `.dbos` folder? Won't `getCloudCredentials` fail to load the credentials and complain the user is not logged in?

Reply:
Yes--which is fine. The idea is to allow you to run the commands from another app's directory (as you likely only have the cloud CLI installed in app directories, you can't run these commands from random directories until we make it work globally, which we haven't yet).

Reply:
it might be misleading for users to do a login and spread `.dbos` folder everywhere in their system

Reply:
Will fix in a future PR.